types/types.go

```diff
@@ -31,6 +34,14 @@ type Device struct {
 	NewestEvents      *Event  `json:"newest_events"`
 }
 
+func (d Device) IsRemo() bool {
```

Comment:
Can you think of a better method name here? Since basically I think everything "Is a Remo", "IsRemo" seems a bit obscure. But I can't think of a better name either... Maybe IsRemoFirmware?
If there isn't a better name, I think we can leave this as is. Just caught my eye a little.

Reply:
I don't want to know the firmware version.
I think the name is ok because I want to know if this device is a REMO or not.

However, as commented below, this method may not be necessary.